controllers/api/v1alpha1/cfserviceinstance_types.go

```diff
@@ -49,10 +48,6 @@ type InstanceType string
 
 // CFServiceInstanceStatus defines the observed state of CFServiceInstance
 type CFServiceInstanceStatus struct {
-	// A reference to the Secret containing the credentials (same as spec.secretName).
-	// This is required to conform to the Kubernetes Service Bindings spec
-	Binding v1.LocalObjectReference `json:"binding"`
```

Comment:
This is required to meet the "ProvisionedService" duck type which lets us interop with other tooling in the "K8s Service Binding" ecosystem. Kpack, the Service Binding reconciler that volume mounts service bindings as secrets, etc. all require this when you use them in certain ways.

https://servicebinding.io/spec/core/1.0.0/#provisioned-service

Those projects support working with anything that has a `spec.binding.status` field which is why we have this.